models/rule_helpers.go

```diff
@@ -0,0 +1,228 @@
+	Disabled     bool
+	RuleParams   RuleParams
+	For          *duration.Duration
+	Severity     managementpb.Severity
+	CustomLabels map[string]string
+	Filters      Filters
+	ChannelIDs   []string
+}
+
+// CreateRule persists alert Rule.
+func CreateRule(q *reform.Querier, params *CreateRuleParams) (*Rule, error) {
+	id := "/rule_id/" + uuid.New().String()
+
+	if err := checkUniqueRuleID(q, id); err != nil {
+		return nil, err
+	}
+
+	row := &Rule{
+		ID:       id,
+		Template: makeModelTemplate(params),
+		Disabled: params.Disabled,
+		For:      params.For.AsDuration(),
+		Severity: params.Severity.String(),
+		Filters:  params.Filters,
+	}
+
+	labels, err := json.Marshal(params.CustomLabels)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to create alert rule")
+	}
+	row.CustomLabels = labels
+	row.Params = params.RuleParams
+
+	channels := make(Channels, len(params.ChannelIDs))
+	for i, cid := range params.ChannelIDs {
+		ch, err := FindChannelByID(q, cid)
+		if err != nil {
+			return nil, errors.WithMessage(err, "failed to create alert rule")
+		}
+		channels[i] = ch
+	}
+	row.Channels = channels
+
+	err = q.Insert(row)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to create alert rule")
+	}
+
+	return row, nil
+}
+
+// UpdateRuleParams is params for updating existing Rule.
+type UpdateRuleParams struct {
+	RuleID       string
+	Disabled     bool
+	RuleParams   RuleParams
+	For          *duration.Duration
+	Severity     managementpb.Severity
+	CustomLabels map[string]string
+	Filters      Filters
+	ChannelIDs   []string
+}
+
+// UpdateRule updates existing alerts Rule.
+func UpdateRule(q *reform.Querier, RuleID string, params *UpdateRuleParams) (*Rule, error) {
+	row, err := FindRuleByID(q, RuleID)
+	if err != nil {
+		return nil, err
+	}
+
+	row.Disabled = params.Disabled
+	row.For = params.For.AsDuration()
+	row.Severity = params.Severity.String()
+	row.Filters = params.Filters
+	row.Params = params.RuleParams
+
+	labels, err := json.Marshal(params.CustomLabels)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to update alert rule")
+	}
+	row.CustomLabels = labels
+
+	channels := make(Channels, len(params.ChannelIDs))
+	for _, cid := range params.ChannelIDs {
+		ch, err := FindChannelByID(q, cid)
+		if err != nil {
+			return nil, errors.WithMessage(err, "failed to update alert rule")
+		}
+		channels = append(channels, ch)
+	}
+	row.Channels = channels
+
+	err = q.Update(row)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to update alerts Rule")
+	}
+
+	return row, nil
+}
+
+// RemoveRule removes alert Rule with specified id.
+func RemoveRule(q *reform.Querier, id string) error {
+	if _, err := FindRuleByID(q, id); err != nil {
+		return err
+	}
+
+	err := q.Delete(&Rule{ID: id})
+	if err != nil {
+		return errors.Wrap(err, "failed to delete alert Rule")
+	}
+	return nil
+}
+
+// TODO remove once template API is merged
+func makeModelTemplate(params *CreateRuleParams) *Template {
```

Comment:
writing a dummy template to the DB for now, will use template model helpers once 6909 is merged